src/sage/arith/misc.py

```diff
@@ -1955,27 +1956,41 @@ def xlcm(m, n):
     return (l, m, n)
 
 
-def xgcd(a, b):
+def xgcd(a, b=None):
     r"""
-    Return a triple ``(g,s,t)`` such that `g = s\cdot a+t\cdot b = \gcd(a,b)`.
+    When both ``a`` and ``b`` are given, then return a triple ``(g,s,t)``
```

Comment:
```suggestion
    Return the greatest common divisor and the Bézout coefficients of the input arguments.

    When both ``a`` and ``b`` are given, then return a triple ``(g,s,t)``
```
The first line should be a one sentence summary of the algorithm, I believe.